Lab1/app/routes/route_test.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint
+
+from app.models.User import User
+
+test_blueprint = Blueprint('route_test', __name__, url_prefix='/test')
```

Comment:
We can use this Blueprint object to create another app that handles request starting with url_prefix'/test'.

This 'test_blueprint' app will then be added to the main app using the code above.